packages/xod-tabtest/src/Tabtest.re

```diff
@@ -266,9 +266,15 @@ let generateSuite = (project, patchPathToTest) : XResult.t(t) => {
   | (Some(patchUnderTest), Some(tsv)) =>
     let bench = Bench.create(project, patchUnderTest);
     let probes = bench.probes;
-    let benchPatchPath = "@/tabtest-bench";
+    let benchPatchPath = "tabtest-" ++ patchPathToTest;
+    let safeBasename =
+      PatchPath.getBaseName(patchPathToTest)
+      |> Js.String.replace("(", "__")
+      |> Js.String.replace(")", "");
```

Comment:
What about patches with more than one generic type(`foo(number,string)` etc)?

Reply:
Oops, commas should also be replaced. Will fix it now.